services/src/main/java/org/fao/geonet/api/AllRequestsInterceptor.java

```diff
@@ -42,24 +43,18 @@
  * crawlers.
  */
 public class AllRequestsInterceptor extends HandlerInterceptorAdapter {
+
     /**
      * List of bots to avoid.
      */
-    public static final String BOT_REGEXP = ".*(bot|crawler|baiduspider|80legs|ia_archiver|"
-        + "voyager|yahoo! slurp|mediapartners-google).*";
+    @Value("${bot.regexpfilter}:.*(bot|crawler|baiduspider|80legs|ia_archiver|voyager|yahoo! slurp|mediapartners-google).*")
```

Comment:
Invalid syntax - } in the wrong place.
@Value("${bot.regexpfilter:.*(bot|crawler|baiduspider|80legs|ia_archiver|voyager|yahoo! slurp|mediapartners-google).*}")